RetailCoder.VBE/Navigation/CodeExplorer/CodeExplorerViewModel.cs

```diff
@@ -552,46 +558,15 @@ private void ExecuteRemoveComand(object param)
 
         private bool CanExecuteExportAllCommand => ExportAllCommand.CanExecute(SelectedItem);
 
-        public Visibility ExportVisibility
-        {
-            get
-            {
-                if (CanExecuteExportAllCommand == false)
-                { return Visibility.Visible; }
-                else { return Visibility.Collapsed; }
-            }
-        }
+        public Visibility ExportVisibility => CanExecuteExportAllCommand == false ? Visibility.Visible : Visibility.Collapsed;
```

Comment:
we have a `BoolToVisibilityConverter`. Should we use that one instead?

Reply:
Where is that found? I did a search for it but didn't find one.

Reply:
The logic on this one is opposite `ExportAllVisibility` below. Is that correct? Rewriting it without the ` == false` leads to `public Visibility ExportVisibility => CanExecuteExportAllCommand ? Visibility.Collapsed : Visibility.Visible;` or am I mistaken?

Reply:
found [here](https://github.com/rubberduck-vba/Rubberduck/tree/next/RetailCoder.VBE/UI/Converters)

Reply:
Can this be done on properties? Everything I understood was dealing with XAML and going through the binding engine.

Reply:
you need to pass the converter to the Binding. Something like `Visibility={Binding Path=ExportVisibility, Converter={StaticResource BoolToVisibilityConverter}}` if I'm going by how i've seen it

Reply:
Are lines https://github.com/IvenBach/Rubberduck/blob/606b60eefbe8459fc27e8cd8f6b8d999b8e82e61/RetailCoder.VBE/UI/CodeExplorer/CodeExplorerControl.xaml#L234, https://github.com/IvenBach/Rubberduck/blob/606b60eefbe8459fc27e8cd8f6b8d999b8e82e61/RetailCoder.VBE/UI/CodeExplorer/CodeExplorerControl.xaml#L238, and https://github.com/IvenBach/Rubberduck/blob/606b60eefbe8459fc27e8cd8f6b8d999b8e82e61/RetailCoder.VBE/UI/CodeExplorer/CodeExplorerControl.xaml#L528 correct usage of the converter?